akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/ProtoAnySerialization.scala

```diff
@@ -132,19 +159,37 @@ import scalapb.options.Scalapb
         encode(msg)
       case msg: GeneratedMessageV3 =>
         encode(msg)
+      case SerializedEvent(bytes, id, manifest) =>
+        if (id == akkaProtobufSerializer.identifier && isDefaultAkkaProtobufSerializer) {
+          // see corresponding typeUrl cases in `toSerializedEvent`
+          allReverseTypes.get(manifest) match {
+            case Some(typeName) =>
+              val typeUrl = GoogleTypeUrlPrefix + typeName
+              ScalaPbAny(typeUrl, ByteString.copyFrom(bytes))
+            case None =>
+              val errMsg =
+                if (allDescriptors.isEmpty)
+                  s"Unable to find Protobuf descriptor for message class: [$manifest]. " +
+                  "No descriptors defined when creating GrpcReadJournal. Note that GrpcReadJournal should be created " +
+                  "with the GrpcReadJournal apply/create factory method and not from configuration via GrpcReadJournalProvider " +
+                  "when using Protobuf serialization."
+                else
+                  s"Unable to find Protobuf descriptor for class: [$manifest]."
+              throw new IllegalArgumentException(errMsg)
+          }
+        } else {
+          // Akka serialization
+          val typeUrl = akkaSerializationTypeUrl(id, manifest)
+          ScalaPbAny(typeUrl, ByteString.copyFrom(bytes))
```

Comment:
yes, I know that discussion, but I think the conclusion then and now is that "we don't care" because the Akka serialization scheme is only valid if Akka on both sides, which knows about this trick.

Alternative would be to extend the `message Event` in the protocol with a specific AkkaSerializedPayload. Anyway, if we would change that it would be a separate PR.

Reply:
Yep, I think we should leave it as is, just wanted to get that written down somewhere.